index.js

```diff
@@ -36,14 +36,18 @@ const createRetrieval = async (req, res, client) => {
     INSERT INTO retrievals (
       retrieval_template_id,
       spark_version,
-      zinnia_version
+      zinnia_version,
+      created_at_round,
+      created_from_address
     )
-    VALUES ($1, $2, $3)
+    VALUES ($1, $2, $3, $4, $5)
     RETURNING id
   `, [
     retrievalTemplate.id,
     meta.sparkVersion,
-    meta.zinniaVersion
+    meta.zinniaVersion,
+    round,
+    req.connection.remoteAddress
```

Comment:
_Re-posting https://github.com/filecoin-station/spark-api/pull/56#discussion_r1310404604_

> > What are the implications for the privacy of Station's users when we start collecting their IP addresses?
> 
> looks like we need to add a privacy policy https://termly.io/faq/is-ip-address-personal-information-under-ccpa/#:~:text=IP%20addresses%20are%20classified%20as,this%20in%20a%20privacy%20policy., cc @coreymjames

Reply:
Would anything change if we collect only the IPv4 /24 subnet instead of the full IPv4 address? Is the subnet considered as a personal information too?

Reply:
Good question! @coreymjames could you please help us figure out the legal implications?